config/jobs/kubernetes/sig-auth/serviceaccount-admission-controller-migration-config.yaml

```diff
@@ -0,0 +1,23 @@
+periodics:
+- name: serviceaccount-admission-controller-migration
+  annotations:
+    testgrid-dashboards: sig-auth-gce
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  decorate: true
+  interval: 2h
+  spec:
+    containers:
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20200918-58bdf8d-master
+    - args:
+      - --timeout=920
+      - --bare
+      - --scenario=kubernetes_e2e
```

Comment:
is this testing an upgrade or just a regular e2e test?

Reply:
it is a upgrade test even though there is no upgrade_args specified.

since `--test_args="--ginkgo.focus=\[Feature:BoundServiceAccountTokenVolume\]"` will select the upgrade test for this feature, it will run the upgrade test framework. the first target would be `--extract=ci/latest-1.19` and the unspecified target would be `ci/latest`.

Reply:
I see... can we make the upgrade target more explicit in the job config?

Reply:
done